core/api/views/cp_archive.py

```diff
@@ -113,16 +113,25 @@ def get_usages(self, cp_report):
                 description="Country programme report archive id",
                 type=openapi.TYPE_INTEGER,
             ),
+            openapi.Parameter(
+                "convert_data",
+                openapi.IN_QUERY,
+                description="Convert values to ODP tonnes (SectionA) Co2-eq (SectionB)",
+                type=openapi.TYPE_BOOLEAN,
+            ),
         ],
     )
     def get(self, *args, **kwargs):
+        convert_data = str(self.request.query_params.get("convert_data", None))
         cp_report = self._get_cp_report()
         if cp_report.year > IMPORT_DB_MAX_YEAR:
             exporter = CPReportNewExporter(cp_report)
         else:
             exporter = CPReportOldExporter(cp_report)
 
-        wb = exporter.get_xlsx(self.get_data(cp_report), self.get_usages(cp_report))
+        wb = exporter.get_xlsx(
+            self.get_data(cp_report), self.get_usages(cp_report), convert_data == "1"
```

Comment:
just nitpicking, but I would use a variable defined before the `get_xlsx` statement rather than an equality check as a function param:
```
use_full_history = convert_data == "1"
wb = exporter.get_xlsx(
            self.get_data(cp_report), self.get_usages(cp_report), use_full_history
)
```